honeybadger/connection.py

```diff
@@ -49,6 +52,40 @@ def send_notice(config, payload):
     return notice_id
 
 
-def send_event(config, payload):
-    path = "/v1/events/"
-    return _make_http_request(path, config, payload)
+def send_events(config, payload) -> EventsSendResult:
+    """
+    Send events synchronously to Honeybadger.
```

Comment:
I guess the reason this is synchronous is because it is intended to be called from the events worker, right? We could point that out here, just in case.